database-plugins/src/main/java/io/cdap/plugin/db/common/DBBaseConfig.java

```diff
@@ -144,8 +144,4 @@ public String getReferenceName() {
   public String getColumnNameCase() {
     return columnNameCase;
   }
-
-  public boolean getUseConnection() {
-    return useConnection == null ? false : useConnection;
-  }
```

Comment:
why is this removed?

Reply:
Because it was a placeholder to default connection to false, now since we reenable it, we don't need the default value any more